app/assets/stylesheets/components/_post.scss

```diff
@@ -107,6 +121,15 @@ $Post-img-max-width: 800px + $Theme-spacing-small * 2;
       white-space: pre-wrap;
       word-wrap: break-word;
     }
+
+    code {
+      @include font-properties(code, base, $Theme-font-sizes-extra);
+
+      padding-top: 0;
+      padding-bottom: 0;
+
+      background-color: transparent;
```

Comment:
To override what we're setting on `code`. The thing is that there is no real good way of differentiating between inline and block code, other than block has a parent that is `.highlight`, so I set the style for the inline on `code` and undid some of it on `.highlight code`.